### DISCUSSION TITLE
[vulkan] port for minimal sdk

### DISCUSSION THREAD

**Discussion by pheonixfirewingz** [microsoft/vcpkg#25157] [Q&A]
I know in the vulkan port says this
```cmake 
# Due to the complexity involved, this package doesn't install the Vulkan SDK.
# It instead verifies that Vulkan is installed.
# Other packages can depend on this package to declare a dependency on Vulkan.
```
but is that due to no one want to work on it or it would not be accepted by the rules

i'm planning on squashing together in on port:
https://github.com/microsoft/vcpkg/blob/master/ports/vulkan
https://github.com/microsoft/vcpkg/blob/master/ports/vulkan-headers
https://github.com/microsoft/vcpkg/blob/master/ports/vulkan-hpp
https://github.com/KhronosGroup/Vulkan-Loader
https://github.com/KhronosGroup/Vulkan-ValidationLayers

would this get merge or would I waste my time?

**Comment by gracicot** [comment]
That would actually be really useful for me. I'm trying to get google dawn to work with vcpkg but it turns out it's quite hard without all of those packages.
